pkg/kapp/resources/matchers.go

```diff
@@ -124,17 +124,20 @@ func (m HasNamespaceMatcher) Matches(res Resource) bool {
 }
 
 var (
+	// TODO should we just generically match *.k8s.io?
 	builtinAPIGroups = map[string]struct{}{
 		"":                             struct{}{},
 		"admissionregistration.k8s.io": struct{}{},
 		"apiextensions.k8s.io":         struct{}{},
+		"apiregistration.k8s.io":       struct{}{},
 		"apps":                         struct{}{},
 		"authentication.k8s.io":        struct{}{},
 		"authorization.k8s.io":         struct{}{},
 		"autoscaling":                  struct{}{},
 		"batch":                        struct{}{},
 		"certificates.k8s.io":          struct{}{},
 		"coordination.k8s.io":          struct{}{},
+		"discovery.k8s.io":             struct{}{},
 		"extensions":                   struct{}{},
```

Comment:
We are still missing some of them. e.g. core, events.k8s.io,  flowcontrol.apiserver.k8s.io and internal.apiserver.k8s.io(ref: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#-strong-api-groups-strong-). Any reason not to include them ? 
--


<br class="Apple-interchange-newline">
--


<br class="Apple-interchange-newline">

Reply:
updated.